src/Helpers/hyperlink.helper.ts

```diff
@@ -0,0 +1,34 @@
+import * as mongoose from 'mongoose';
+
+import Section from '../Interfaces/model/section.interface';
+import Notebook from '../Interfaces/model/notebook.interface';
+import hyperlinkModel from '../Models/hyperlink.model';
+
+/**
+ * Create a hyperlink
+ * @param {string} key
+ * @param {Section} link
+ * @param {Notebook} notebook
+ */
+export const addHyperlink = async (key: string, link: Section, notebook: Notebook) => {
+    const hyperlink = await hyperlinkModel.create({ key: key, link: link, notebook: notebook });
+    notebook.links.push(hyperlink);
```

Comment:
can we do this? shouldn't we find the document from db and add it? if it is possible then @Pavi114 update the add cell to section function using this

Reply:
We can do this because I changed the interface to extend a document.